price_feed_poller.py

```diff
@@ -172,19 +222,41 @@ def main(args):
     gas = config["network"].get("gas", 4000000)
     # Get gas price, defaults to 100 gwei:
     gas_price = config["network"].get("gas_price", 100000000000) 
+    # Get HTTP-JSON-RPC waiting timeout (in secs):
+    tx_waiting_timeout_secs = config["network"].get("tx_waiting_timeout_secs", 120) 
+    # Get HTTP-JSON-RPC polling latency timeout (in secs):
+    tx_polling_latency_secs = config["network"].get("tx_polling_latency_secs", 5) 
```

Comment:
I wonder if the best default value for this would be that of the Ethereum average block period. The value used in `loop_interval_secs` seems right.

Reply:
Average block time:

- Kovan: 4 seconds
- Rinkeby: 15 seconds
- Goërli: 15 seconds (avg)
- Ropsten: < 30 seconds
- Homestead: 13 seconds ([avg](https://ycharts.com/indicators/ethereum_average_block_time#:~:text=Ethereum%20Average%20Block%20Time%20is,0.45%25%20from%20one%20year%20ago.))

Default value for new parameter "tx_polling_latency_secs", if none is specified in config file: 13